### PR TITLE
payment checkout page added

### PR DESCRIPTION
**Pull request by vaibhav18matere**

### CODE REVIEW

**Review by shaksham08 — APPROVED** [review]
Looks Good to me

File: checkout/checkout.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+.selection select{
+    padding: 0.625rem 1.25rem;
+}
+.payment-cards img{
+    width: 4.5rem;
+    padding: 0.625rem;
+}
+.confirm-btn{
+    border-radius: 24px;
```
Comment (manojasarsa) [inline checkout/checkout.css]: I would suggest to be consistent with units. As you are using "rem", so you can provide border radius in rem as well.

**Review by manojasarsa — APPROVED** [review]
Rest looks good!

File: checkout/checkout.css (hunk 1 above)
Comment (vaibhav18matere) [reply checkout/checkout.css]: will do that, thank you for bringing this to my notice

**Pull request merged**